App/State/EnvironmentState.swift

```diff
@@ -16,7 +16,7 @@ import ImmuniExposureNotification
 import Katana
 import Models
 import UIKit
-import UserNotifications
+import PushNotification
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/State/EnvironmentState.swift;
  Line: 19;
-->

- :warning: swiftlint (0.39.2): Imports should be sorted. (sorted_imports)